easybuild/tools/filetools.py

```diff
@@ -1671,7 +1672,7 @@ def copytree(src, dst, symlinks=False, ignore=None):
     class Error(EnvironmentError):
         pass
     try:
-        WindowsError  # @UndefinedVariable
+        WindowsError = WindowsError # noqa
```

Comment:
I'd rather just remove this whole body (and clean up the docstring to basically just mention "use copy_dir instead".

It's dead code anyway, since the `log.deprecated` will raise an error now we're in EasyBuild 4.x...

Reply:
Good point. BTW: When are those deprecated messages meant to be remove? 2 versions after?

Reply:
The `log.deprecated` resulting in a warning in EasyBuild 3.x, now it results in an error.
So, it should definitely stay for now (to avoid a nasty crash rather than a clear error), but I guess it could be dropped in EasyBuild 5.0.

Reply:
Ok, so 2 versions (or 1 version after "expiry"/version mentioned)